cmd/skywire-cli/commands/visor/transports.go

```diff
@@ -113,15 +113,15 @@ var addTpCmd = &cobra.Command{
 
 			logger.Infof("Established %v transport to %v", transportType, pk)
 		} else {
-			transportTypes := []string{
-				tptypes.STCP,
-				tptypes.STCPR,
-				tptypes.SUDPH,
+			transportTypes := []network.Type{
+				network.STCP,
+				network.STCPR,
+				network.SUDPH,
 				dmsg.Type,
```

Comment:
Imo it would be better to either use `dmsg.Type` everywhere or use `network.DMSG`.